src/fluent-plugin-mdsd/Makefile

```diff
@@ -14,9 +14,11 @@ all: $(MDSDGEM)
 
 $(MDSDGEM): $(MDSDGEM).gemspec lib/fluent/plugin/out_mdsd.rb
 	cp -f ../builddir/release/lib/Liboutmdsdrb.so lib/fluent/plugin/Liboutmdsdrb.so
+	cp -f ../../LICENSE.txt .
+	cp -f ../../README.md .
 	$(GEM) build $(MDSDGEM).gemspec
```

Comment:
This really bothers me. The gem depends on Liboutmdsdrb.so but has no instructions to build it. It includes the LICENSE.txt and README.md files but does not depend on them. If any of those files are altered, but the .gemspec and out_mdsd.rb files are not, running "make" in this directory won't do anything.